doc/Contribute.md

```diff
@@ -89,17 +89,14 @@ associated documentation, if we have any.
 |[__top__/dev/cf_cli](cf_cli.md)                                        |Deploy cf cli into a helper pod from which to then     |
 |                                                                       |inspect the deployed Kubecf                            |
 |[__top__/dev/kube](inspection.md)                                      |Tools to inspect kube clusters and kubecf deployments. |
-|[__top__/dev/linters](linters.md)                                      |Tools for statically checking the kubecf sources.      |
```

Comment:
This appears to still exist.  Not sure it's _worth_ existing, but it's there for now.

Many of the other linked documents also still exist.

Reply:
True. I'm happy leaving bazel things around until its scheduled removal, but wanted only 1 entry point for newcomers. Afterall, is the task of the card.

Reply:
(fwiw removal is scheduled in https://github.com/cloudfoundry-incubator/kubecf/issues/1011)